### PR TITLE
Many villager-related mappings, some trade related ones and some new …

### PR DESCRIPTION
**Pull request by liach**
…block related ones (two furnaces)

Also mapped some lost entity methods.

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Comment by asiekierka** [comment]
Some areas overlap with https://github.com/FabricMC/yarn/pull/278 - please discuss that over.

**Comment by Prospector** [comment]
I'd say accept any conflicts from liach's files over mine because he did a deeper dive into villagers and probably has a better understanding than I do as I didn't focus much on them.

**Comment by asiekierka** [comment]
It'd be appreciated if you could remove the conflicts from yours.first

**Comment by liach** [comment]
Any more issues?

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping
Comment (shadowfacts) [inline mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: Is willing to do what?

File: mappings/net/minecraft/village/RaidState.mapping
Comment (shadowfacts) [inline mappings/net/minecraft/village/RaidState.mapping]: Should be in `net/minecraft/village/raid` like `Raid`.

**Review by shadowfacts — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping
Comment (asiekierka) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: That's the field name. But yes, we don't need to match them exactly...

File: mappings/net/minecraft/village/RaidState.mapping
Comment (asiekierka) [reply mappings/net/minecraft/village/RaidState.mapping]: +1

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping
Comment (liach) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: i think it is for mating.

**Comment by liach** [comment]
Anyways, I've done the conflict resolution and now things should be fine. Raid and RaidState classes are fully mapped. (method names and field names)

**Review by shadowfacts — APPROVED** [review]

File: mappings/net/minecraft/client/render/entity/VillagerEntityRenderLayer.mapping
Comment (asiekierka) [inline mappings/net/minecraft/client/render/entity/VillagerEntityRenderLayer.mapping]: Nuh-uh! That's a static final!

File: mappings/net/minecraft/village/VillagerTrades.mapping
Comment (asiekierka) [inline mappings/net/minecraft/village/VillagerTrades.mapping]: nuh-uh! I'm a static final too!

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/entity/VillagerEntityRenderLayer.mapping
Comment (liach) [reply mappings/net/minecraft/client/render/entity/VillagerEntityRenderLayer.mapping]: I thought a static final field was not necessarily immutable, will change when I get back on enigma https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Comment (asiekierka) [reply mappings/net/minecraft/client/render/entity/VillagerEntityRenderLayer.mapping]: Our convention to date was to call all static finals UPPER_CASE. If you want to change *convention*, that's probably a new issue altogether

**Comment by Prospector** [comment]
I'm going to wait for this to be merged before I try to resolve conflicts with mine

**Comment by liach** [comment]
Fixed the names for 3 static final fields I could find.

**Review by modmuss50 — APPROVED** [review]
Looks good 👍

**Comment by modmuss50** [comment]
I will wait for @asiekierka to be happy with before it gets merged

**Comment by liach** [comment]
You can merge other prs first. I can fix this one later.

**Review by Player3324 — APPROVED** [review]
No obvious issues

**Comment by liach** [comment]
@asiekierka Can you review again?

File: mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping
Comment (asiekierka) [inline mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping]: Modelled? Not Modeled?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping
Comment (jamierocks) [reply mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping]: modelled is correct I believe

Comment (asiekierka) [reply mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping]: In BrE, yes, but I think our mappings *generally* assume AmE?

Comment (Prospector) [reply mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping]: yup. It should be Modeled

Comment (liach) [reply mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping]: Resolved.

**Comment by liach** [comment]
Thx!

**Pull request closed without merging**
